yaml/pois.yaml

```diff
@@ -1619,11 +1619,17 @@ filters:
       <<: *output_properties
       kind: {col: amenity}
   - filter:
-      amenity: [bus_station, car_rental, car_sharing, recycling, shelter]
+      amenity: [bus_station, car_rental, recycling, shelter]
     min_zoom: 16
     output:
       <<: *output_properties
       kind: {col: amenity}
+  - filter:
+      amenity: car_sharing
+    min_zoom: 19
```

Comment:
We should treat this more like we do `bicycle_parking`, also to abide by SemVer so the default for names only changes one zoom (16 to 17), but then the no-names can go to zoom 19.

```
  - filter:
      amenity: bicycle_parking
      name: true
    min_zoom: 17
    output:
      <<: *output_properties
      <<: *cycle_parking_properties
      kind: bicycle_parking
  - filter:
      amenity: bicycle_parking
    min_zoom: 19
    output:
      <<: *output_properties
      <<: *cycle_parking_properties
      kind: {col: amenity}
```